pepper-and-carrot-ep6.msrt

```diff
@@ -699,7 +699,7 @@
 [eng] Incredible!
 [epo] Nekredeble!
 [jbo] .i spaji
-[fra] Incroyable !
+[fra] Inoui !
```

Comment:
You are right about the accent. I thought the 1990 reform removed it.
This word is visualised from 4:04:06 to 4:04:22 hence 700ms with 3 lip moves (including a short one).
To hear it clearly, even by switching to a different way of talking from the mayor, from a long theatrical one to a faster colloquial one, the word "incroyable" doesn't fit nicely in my opinion.

"Inouï" allows to remain the same pace from the previously exclamation and sticks with the 700ms timeframe.